package.json

```diff
@@ -5,18 +5,24 @@
   "license": "MIT",
   "author": "Bjorn Lu",
   "bin": {
-    "create-vite-extra": "index.js"
+    "create-vite-extra": "./dist/index.js"
+  },
+  "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsc index.ts",
+    "format": "prettier --write .",
+    "update-deps": "pnpm update -r --latest && node update-deno-deps.js"
   },
   "files": [
-    "index.js",
+    "index.ts",
```

Comment:
This like needs to be `dist` instead. The `main` field below also needs updating